src/SpecialCreateBranch.php

```diff
@@ -78,29 +80,17 @@ public function onSubmit( array $data ) {
 		// create branch root page
 		$page = MediaWikiServices::getInstance()->getWikiPageFactory()->newFromTitle( $title );
 
-		$text = "This is branch root page. It is autogenerated on branch creation.\n";
-		$text .= "{| class=\"wikitable\"\n";
-		$text .= "|+ Branch parameters\n";
-		$text .= "|-\n";
-		$text .= sprintf( "| Id || %s\n", $branchId );
-		$text .= "|-\n";
-		$text .= sprintf( "| Name || %s\n", $data['BranchName'] );
-		$text .= "|-\n";
-		$text .= sprintf( "| Token || %s\n", $data['Token'] );
-		$text .= "|-\n";
-		$text .= sprintf( "| Type || %s\n", $data['BranchType'] );
-		$text .= "|-\n";
-		$text .= sprintf( "| Allow conflicts || %s\n", $data['AllowConflicts'] ? 'yes' : 'no' );
-		$text .= "|}\n";
-		$content = ContentHandler::makeContent( $text, $title );
+		$json = FormatJson::encode( [ 'BranchId' => $branchId ] + $data );
+		$content = ContentHandler::makeContent( $json, $title, CONTENT_MODEL_JSON );
```

Comment:
So the process for storing thing in json here is: First to encode, then to use the ContentHandler and  that will write it to storage?